tests/tests/fv3_gfs_v16_RRTMGP

```diff
@@ -38,6 +44,12 @@ export LIST_FILES="atmos_4xdaily.tile1.nc \
                    dynf024.tile4.nc \
                    dynf024.tile5.nc \
                    dynf024.tile6.nc \
+                   dynf048.tile1.nc \
```

Comment:
This won't work - how do you want to compare 48 output files if the runtime is only 24 hours?

Reply:
Sorry, I added +export FHMAX=48 +export RESTART_INTERVAL=24 to make the GP v16 RTs just as for RRTMG.

Reply:
Does the job still complete within 30 minutes?

Reply:
yes